roles/variables/vars/main.yml

```diff
@@ -209,12 +209,10 @@ kafka_controller_properties:
   metrics_reporter:
     enabled: "{{ kafka_controller_metrics_reporter_enabled|bool }}"
     properties:
-      metric.reporters: io.confluent.metrics.reporter.ConfluentMetricsReporter
-      confluent.metrics.reporter.bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | confluent.platform.resolve_hostnames(hostvars) | join(':' + kafka_broker_listeners['internal']['port']|string + ',') }}:{{kafka_broker_listeners['internal']['port']}}"
-      confluent.metrics.reporter.topic.replicas: "{{kafka_broker_default_internal_replication_factor}}"
+      confluent.metrics.reporter.bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | confluent.platform.resolve_hostnames(hostvars) | join(':' + kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['port']|string + ',') }}:{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['port']}}"
```

Comment:
Looks good. Just to confirm, you don't need these properties in Controller ?
metric.reporters
confluent.metrics.reporter.topic.replicas

Reply:
yes that is under discussion, I will confirm once and then update this accordingly

Reply:
Hi @nsharma-git , I confirmed it with kafka team and have updated the configs. Please review